web/client-api/src/main/java/io/deephaven/web/client/api/tree/JsTreeTable.java

```diff
@@ -1097,6 +1223,16 @@ public JsArray<FilterCondition> getFilter() {
         return JsItr.slice(filters);
     }
 
+    /**
+     * The current list of custom columns added to this Tree Table.
+     *
+     * @return {@link CustomColumn} array
+     */
+    @JsProperty
+    public JsArray<CustomColumn> getUpdateColumns() {
```

Comment:
This should be getCustomColumns() to match Table.

And if we have to go one more round anyway... in the class javadoc it says this:
```
* <li>Custom columns are not directly supported. If the TreeTable was created client-side, the original Table can have
* custom columns applied, and the TreeTable can be recreated.</li>
```
can you update that to say that custom columns are supported for rollups, but not trees?